project_event/models/project_task.py

```diff
@@ -1009,6 +1050,22 @@ def is_hr_resource_booked(self, attendee):
         ])
         return len(overlaps_partners) > 0
 
+    def is_hr_resource_double_booked(self, attendee, date_start, date_end):
```

Comment:
For the method is_hr_resource_booked, i forget to check if it is not the same event_task_id, so when added we have to merge the 2 methods and keep only one